core/sql/exp/exp_conv.cpp

```diff
@@ -4280,6 +4341,9 @@ unicodeToSByteTarget(
        if ( allowInvalidCodePoint == FALSE )
        {
           ExRaiseSqlError(heap, diagsArea, EXE_INVALID_CHAR_IN_TRANSLATE_FUNC);
+          char hexstr[256];
+          memset(hexstr,0,256);
```

Comment:
The memset is unneeded since stringToHex also does a memset, but it doesn't hurt any.